### DISCUSSION TITLE
Clojure/joyride code as quickpick input for tasks?

### DISCUSSION THREAD

**Discussion by jwhitlark** [BetterThanTomorrow/calva#2122] [Ideas]
I use [task-shell-input](https://marketplace.visualstudio.com/items?itemName=augustocdias.tasks-shell-input) to enumerate my babashka tasks, to run via the vscode/task interface.  Is there interest/need to be able to do something similar with clojure code? 

Here's what a configuration looks like:
```json
"inputs": [
    {
      "id": "bbTasks",
      "type": "command",
      "command": "shellCommand.execute",
      "args": {
        "command": "bb tasks | tail -n +3",
        "cwd": "${workspaceFolder}",
        "fieldSeparator": "|",
        "useSingleResult": false, 
        "taskId": "jw_bb_last_task_run",
        "rememberPrevious": true,
        "env": {
          "WORKSPACE": "${workspaceFolder[0]}",
          "PROJECT": "${workspaceFolderBasename}"
        }
      }
    },
  ],
  "tasks": [
    {
      "label": "Run Babashka Task",
      "type": "shell",
      "command": "bb run ${input:bbTasks}",
      "problemMatcher": [], 
      "group": "none",
      "presentation": {
        "reveal": "always",
        "focus": true,
        "panel": "dedicated",
      },
    },
```

It looks something like this:
![Screen Shot 2023-03-20 at 12 38 59 PM](https://user-images.githubusercontent.com/59580/226448210-6d6ff6f4-53b9-4e6e-ad1e-ec564bbd5992.png)

**Comment by PEZ** [comment]
Sounds interesting! I don't quite understand the example. How is that configuration creating those quickpick options?
